api/metrics/views.py

```diff
@@ -104,8 +104,6 @@ def post(self, request, *args, **kwargs):
         Caution - this could be slow if a very large query is executed, so use with care!
         """
         search = self.metric.search()
-        query = request.data.get('query')
```

Comment:
Did you mean to add this to the logs pr?

Reply:
opps!